modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts

```diff
@@ -52,7 +52,7 @@ const renderStatusbar = (editor: Editor, providersBackstage: UiFactoryBackstageP
 
   const getResizeType = (editor): ResizeTypes => {
     // If autoresize is enabled, disable resize
-    const fallback = !Strings.contains(editor.settings.plugins, 'autoresize');
+    const fallback = !Strings.contains(editor.getParam('plugins', '', 'string'), 'autoresize');
```

Comment:
Starting to think we should really have an API to check if a plugin is enabled... I'll make a note for myself to Jira it, if it isn't already.